express/code/blocks/frictionless-quick-action/frictionless-quick-action.js

```diff
@@ -264,21 +265,59 @@ async function uploadAssetToStorage(file) {
   });
 
   progressBar.setProgress(100);
+
+  // Clear upload state on success
+  uploadInProgress = null;
+
+  // Log video upload success for analytics
+  if (file.type.startsWith('video/')) {
+    const uploadDuration = Date.now() - uploadStartTime;
+    window.lana?.log('Video upload successful', {
+      tags: 'frictionless-video-upload-success',
+      assetId: asset.assetId,
+      fileSize: file.size,
+      fileType: file.type,
+      quickAction,
+      uploadDuration,
```

Comment:
These are not valid option parameters for LANA and will be ignored. You should append these strings to the error message itself instead. 
e.g.
```
window.lana?.log('Video upload successful ' + 'id¶' + asset.assetId + ' size¶' + file.size +..., { options-array })
```
You can see the options defaults here: https://github.com/adobecom/milo/blob/main/libs/utils/lana.js#L6-L16

You can also refer to https://github.com/adobecom/milo/blob/main/libs/utils/lana.md for examples and https://wiki.corp.adobe.com/display/WCMSOps/LANA+-+Log+Always+Never+Assume#LANALogAlwaysNeverAssume-Parameters for more verbose descriptions.